sources/src/main/java/com/google/solutions/jitaccess/core/catalog/project/AssetInventoryRepository.java

```diff
@@ -138,16 +140,16 @@ public SortedSet<ProjectId> findProjectsWithRequesterPrivileges(
   }
 
   @Override
-  public RequesterPrivilegeSet<ProjectRoleBinding> findRequesterPrivileges(
-      UserId user,
-      ProjectId projectId,
-      Set<ActivationType> typesToInclude,
-      EnumSet<RequesterPrivilege.Status> statusesToInclude) throws AccessException, IOException {
+  public @NotNull RequesterPrivilegeSet<ProjectRoleBinding> findRequesterPrivileges(
+      @NotNull UserEmail user,
+      @NotNull ProjectId projectId,
+      @NotNull Set<ActivationType> typesToInclude,
+      @NotNull EnumSet<RequesterPrivilege.Status> statusesToInclude) throws AccessException, IOException {
```

Comment:
Why is this both in the AssetInventoryRepository and the PolicyAnalyzerRepository?

Reply:
From my understanding these classes have the same purpose but utilize two different APIs for queries to GCP.

Reply:
No. This is not related to my changes.